pkg/plugins/plugins.go

```diff
@@ -179,3 +186,37 @@ func Get(name, imp string) (*Plugin, error) {
 func Handle(iface string, fn func(string, *Client)) {
 	extpointHandlers[iface] = fn
 }
+
+// GetAll returns all the plugins for the specified implementation
+func GetAll(imp string) ([]*Plugin, error) {
+	pluginNames, err := Scan()
```

Comment:
the only nit I have is this scan and activation of all plugins prior to checking if they implement what we're interested in.. I guess there's no solution right now.. (maybe using subdirs for specific plugins - `/volumes`, `/network` to narrow down the list? :smile: not related to this PR only)